tailwind.config.js

```diff
@@ -7,7 +8,8 @@ const bloomTheme = cloneDeep(require("@bloom-housing/ui-components/tailwind.conf
 
 // tailwind will automatically purge unused styles when `NODE_ENV` is set to `production`
 bloomTheme.purge = [
-  __dirname + '/app/javascript/**/*.tsx'
+  "app/javascript/**/*.tsx",
+  "node_modules/@bloom-housing/ui-components/src/**/*.tsx",
```

Comment:
@software-project can you explain how this works?

Reply:
Done